front/style/pages/_grande-marche-europe.scss

```diff
@@ -0,0 +1,286 @@
+.gme {
```

Comment:
This should be in the GME related PR only, no ?

Reply:
I delete gme.scss